packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -366,79 +416,72 @@ export default class GoogleCalendarService implements Calendar {
     }
   }
 
-  async getCacheOrFetchAvailability(args: {
-    timeMin: string;
-    timeMax: string;
-    items: { id: string }[];
-  }): Promise<EventBusyDate[] | null> {
-    const calendar = await this.authedCalendar();
+  async getCacheOrFetchAvailability(args: FreeBusyArgs): Promise<calendar_v3.Schema$FreeBusyResponse> {
     const flags = await getFeatureFlagMap(prisma);
-
-    let freeBusyResult: calendar_v3.Schema$FreeBusyResponse = {};
     if (!flags["calendar-cache"]) {
       this.log.warn("Calendar Cache is disabled - Skipping");
-      const { timeMin, timeMax, items } = args;
-      const apires = await calendar.freebusy.query({
-        requestBody: { timeMin, timeMax, items },
-      });
+      return await this.fetchAvailability(args);
```

Comment:
Wouldn't we then also need to use the `parsedArgs` here? and also below where we are calling the `fetchAvailability` function again

Reply:
Nice catch. Fixing.